requirements/edx/github.txt

```diff
@@ -108,3 +108,5 @@ git+https://github.com/edx/edx-proctoring.git@1.2.3#egg=edx-proctoring==1.2.3
 git+https://github.com/doctoryes/edx-sga.git@c20a8a8e259ef7d5da6394d9f67549200bbb46ee#egg=edx-sga==0.0
 git+https://github.com/open-craft/xblock-poll@7ba819b968fe8faddb78bb22e1fe7637005eb414#egg=xblock-poll==1.2.7
 git+https://github.com/edx-solutions/xblock-drag-and-drop-v2@v2.0.18#egg=xblock-drag-and-drop-v2==2.0.18
+# TODO: Deploy xblock-review to PyPI and pin it before going to master. Talk to Feanil if any questions
+-e git+https://github.com/Dillon-Dumesnil/xblock-review@master#egg=xblock-review
```

Comment:
👍 the above - but definitely 👎 on merging this with a leading `-e`

Reply:
In regards to this comment and the above comment, the xBlock has been pushed to PyPI. Once the other PR is ready for merging, I will push it to PyPI as v1.0.0 and update this install from there so all of the -e / github will be gone